src/node.cc

```diff
@@ -1595,10 +1600,14 @@ static void GetInternalBinding(const FunctionCallbackInfo<Value>& args) {
 
   Local<String> module = args[0].As<String>();
   node::Utf8Value module_v(env->isolate(), module);
+  Local<Object> exports;
 
   node_module* mod = get_internal_module(*module_v);
-  if (mod == nullptr) return ThrowIfNoSuchModule(env, *module_v);
-  Local<Object> exports = InitModule(env, mod, module);
+  if (mod != nullptr) {
+    exports = InitModule(env, mod, module);
+  } else {
+    return ThrowIfNoSuchModule(env, *module_v);
+  }
```

Comment:
Is there a functional difference here?

Reply:
@TimothyGu I was adding the `code_cache` binding to `internalBindnig` but later moved that to `process.binding` for debugging purposes. This is the legacy of that cut-and-paste.